test/diffbinder.cpp

```diff
@@ -0,0 +1,67 @@
+#include <iostream>
```

Comment:
Could you please add author and copyright header

Reply:
Done.

Reply:
👍